lib/omniauth/strategies/bolt.rb

```diff
@@ -60,7 +60,7 @@ def callback_phase
         post_data = URI.encode_www_form(payload)
         res = http.request(req, post_data)
         response = JSON.parse(res.body)
-        @expiration_time = Time.now.utc + 15.minutes
+        @expiration_time = Time.now.utc + response['expires_in']
```

Comment:
Does it assume that these are seconds?

Reply:
yes - it's 3600 which gets automatically interpreted as seconds

Reply:
the payload is 3600
```
[2] pry(main)> Time.now.utc + 3600
=> 2022-06-03 09:17:09.6563547 UTC
[3] pry(main)> Time.now.utc + 3600.seconds
=> 2022-06-03 09:17:12.8559983 UTC
```
also in pure Ruby we can't call `.seconds` on an integer